frontend/ui/src/reducers/data/channels/index.ts

```diff
@@ -36,6 +36,22 @@ const channelsReducer = (state = {}, action: Action): ChannelsState => {
       return action.payload.reduce(setChannel, state);
     case getType(actions.setChannelAction):
       return setChannel(state, action.payload);
+
+    case getType(actions.updateChannelAction):
+      return {
+        ...state,
+        [action.payload.id]: {
+          id: action.payload.id,
+          ...state[action.payload.id],
+          metadata: merge({}, state[action.payload.id]?.metadata, action.payload.metadata),
```

Comment:
I think the `/channels.update` endpoint returns the channel and the complete metadata.
Therefore I think we can use the `addChannelAction` here, but perhaps we should give it a better name. And perhaps you have tested that I am wrong 😄

Reply:
Yeah, but the websocket message does not contain the metadata when i ger the first message from it :)

Reply:
Ah ok I was looking at this usage https://github.com/airyhq/airy/pull/1078/files#diff-b7645a23c2a272bf1fc99bcf3cc2320d846859779044e1190b8bb0efd6247e09R56 

Then I'd perhaps use the `addChannelsAction` there.